backend/battle/tests/test_service.py

```diff
@@ -0,0 +1,88 @@
+from django.test import TestCase
+from model_bakery import baker
+from battle.models import PokemonTeam
+from battle.battles.battle import get_winner_for
+
+
+class GetWinnerTest(TestCase):
+    def setUp(self):
+        super().setUp()
+        self.creator, self.opponent = baker.make('users.User', _quantity=2)
+        battle = baker.make('battle.Battle', creator=self.creator, opponent=self.opponent)
+        self.team_creator = baker.make('battle.Team', battle=battle, trainer=self.creator)
+        self.team_opponent = baker.make('battle.Team', battle=battle, trainer=self.opponent)
+        self.pokemon_1, self.pokemon_2, self.pokemon_3 = baker.make(
+            "pokemon.Pokemon",
+            attack=60, defense=45, hp=50, _quantity=3)
+
+    def test_verify_if_returns_creator_as_winner_when_draw(self):
+        pokemons = [self.pokemon_1, self.pokemon_2, self.pokemon_3]
+        count_creator = 0
+        count_opponent = 0
+        while count_creator < 3:
```

Comment:
I'm not particularly eager to create test info inside while loops, for situations where I really need to I use a for. You can use a for with enumerate

Reply:
The reason is that it makes the intention more clear

Reply:
Thank you for this suggestion!